### PR TITLE
Added about page with test

### PR DESCRIPTION
**Pull request by IssaicBelden**
I added an about model and added an About action in the home controller. I also created a test to check if the about model is a string.

### CODE REVIEW

File: src/TaskIt.Tests/HomeControllerTests.cs [hunk 1]
```diff
@@ -20,6 +20,15 @@ public void IndexReturnsTheDefaultView()
             Assert.IsInstanceOfType(result, typeof(ViewResult));
         }
 
+        [TestMethod]
+        public void AboutReturnsModelWithText()
+        {
+            var controller = new HomeController();
+            var result = controller.About();
+
+            Assert.IsInstanceOfType(result, typeof(string));
```
Comment (JoshuaBelden) [inline src/TaskIt.Tests/HomeControllerTests.cs]: You probably didn't run your tests. :) At least this isn't passing for me. You're expecting an instance of string, but you'll actually be getting an instance of ViewResult.

We can also take this a bit further and make sure that our model is returning as expected:

```
Assert.IsInstanceOfType(result, typeof(ViewResult));

var view = result as ViewResult;
var model = view.Model as AboutViewModel;

Assert.AreEqual(model.SiteDescription, "Your site description here");
```

File: src/TaskIt.Web/Views/Home/About.cshtml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿@model TaskIt.Models.AboutViewModel
```
Comment (JoshuaBelden) [inline src/TaskIt.Web/Views/Home/About.cshtml]: This isn't that big of a deal, but you can change this line to:

```
@model AboutViewModel
```

The reason the page knows about the namespace is from the file '_ViewImports.cshtml' in the root of views. Check it out.

File: src/TaskIt.Tests/HomeControllerTests.cs (hunk 1 above)
Comment (IssaicBelden) [reply src/TaskIt.Tests/HomeControllerTests.cs]: Not sure why, but the test passed for me and failed when I used an integer. Also I see what the assert lines are doing, I don't know what the variables are doing though?

Comment (JoshuaBelden) [reply src/TaskIt.Tests/HomeControllerTests.cs]: I'll start keeping a list of things we should be testing. Unit Testing is pretty straight forward, but there are definitely some best practices.

**Comment by jukeswan** [comment]
Solid parenting. Sweet code, @IssaicBelden!

Hey @JoshuaBelden , FYI, your vortx email account (which now forwards to me) is still associated with your github account, so I'm spying on you like crazy.

**Pull request merged**